server/requestorserver/server.go

```diff
@@ -579,11 +580,11 @@ func (s *Server) checkAuth(w http.ResponseWriter, r *http.Request, rerr *irma.Re
 	if !applies {
 		var ctype = r.Header.Get("Content-Type")
 		if !regexp.MustCompile("^application/json").MatchString(ctype) && !regexp.MustCompile("^text/plain").MatchString(ctype) {
-			s.conf.Logger.Warnf("Session request uses unsupported Content-Type: %s", ctype)
+			s.conf.Logger.Warnf("Session request uses unsupported Content-Type: %s", html.EscapeString(ctype))
 			server.WriteError(w, server.ErrorInvalidRequest, "Unsupported Content-Type: "+ctype)
 			return false
 		}
-		s.conf.Logger.Warnf("Session request uses unknown authentication method, HTTP headers: %s, HTTP POST body: %s", server.ToJson(r.Header), string(body))
+		s.conf.Logger.Warnf("Session request uses unknown authentication method")
```

Comment:
The session request might contain personal data, for instance when having an issuance request. Therefore, logging this can be dangerous. People can still switch to trace mode to see the request message, so we don't have to log that here.